code/controllers/subsystem/mapping.dm

```diff
@@ -53,6 +53,8 @@ SUBSYSTEM_DEF(mapping)
 		if(!config || config.defaulted)
 			to_chat(world, "<span class='boldannounce'>Unable to load next or default map config, defaulting to Box Station</span>")
 			config = old_config
+	GLOB.year_integer += config.year_offset
+	GLOB.announcertype = (config.announcertype == "standard" ? (prob(1) ? "cleanbot" : "classic") : config.announcertype)
```

Comment:
```suggestion
	GLOB.announcertype = (config.announcertype == "Standard" ? (prob(1) ? "medibot" : "classic") : config.announcertype)
```

I'm not going to review all of this right now, but dope.

Reply:
The default value for announcertype in the map's config is `"standard"`, which is case sensitive

Reply:
Well, it wasn't. Now it's.